resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,224 @@
+            </b-img>
+          </b-col>
+          <b-col sm="6" lg="9">
+            <b-input-group>
+              <b-form-input id="application-logo-input"
+                            :placeholder="$t('settings.application.logo.hint')"
+                            v-model="settings.logo"
+                            :disabled="isBusy"
+              >
+              </b-form-input>
+            </b-input-group>
+          </b-col>
+        </b-row>
+      </b-form-group>
+
+      <!--Room limit settings-->
+      <b-form-group
+        label-for="application-room-limit-radio"
+        :description="$t('settings.application.roomLimit.description')"
+      >
+
+        <template v-slot:label>
+          <span><i class="fas fa-person-booth mr-3"></i></span>
+          {{ $t('settings.application.roomLimit.title') }}
+        </template>
+
+        <b-form-radio-group
+          id="application-room-limit-radio"
+          v-model="settings.roomLimit.selected"
+          :options="settings.roomLimit.options"
+          buttons
+          button-variant="success"
+          :disabled="isBusy"
+        >
+        </b-form-radio-group>
+      </b-form-group>
+
+      <!--Pagination page size settings-->
+      <b-form-group
+        label-for="application-pagination-page-size-radio"
+        :description="$t('settings.application.paginationPageSize.description')"
+      >
+
+        <template v-slot:label>
+          <span><i class="fas fa-clone mr-3"></i></span>
+          {{ $t('settings.application.paginationPageSize.title') }}
+        </template>
+
+        <b-form-radio-group
+          id="application-pagination-page-size-radio"
+          v-model="settings.paginationPageSize.selected"
+          :options="settings.paginationPageSize.options"
+          buttons
+          button-variant="success"
+          :disabled="isBusy"
+        >
+        </b-form-radio-group>
+      </b-form-group>
+
+      <!--Own rooms pagination page size settings-->
+      <b-form-group
+        label-for="application-pagination-own-room-page-size-radio"
+        :description="$t('settings.application.ownRoomsPaginationPageSize.description')"
+      >
+
+        <template v-slot:label>
+          <span><i class="fas fa-window-restore mr-3"></i></span>
+          {{ $t('settings.application.ownRoomsPaginationPageSize.title') }}
+        </template>
+
+        <b-form-radio-group
+          id="application-pagination-own-room-page-size-radio"
+          v-model="settings.ownRoomsPaginationPageSize.selected"
+          :options="settings.ownRoomsPaginationPageSize.options"
+          buttons
+          button-variant="success"
+          :disabled="isBusy"
+        >
+        </b-form-radio-group>
+      </b-form-group>
+
+    </b-container>
+  </div>
+</template>
+
+<script>
+import Base from '../../api/base';
+
+export default {
+  data () {
+    return {
+      isBusy: false,
+      settings: {
+        logo: '/images/logo.svg',
+        roomLimit: {
```

Comment:
I would prefer a custom value instead of a select for roomLimit, paginationPageSize and ownRoomsPaginationPageSize

Reply:
https://github.com/THM-Health/PILOS/pull/60/commits/a13a2b68c239d1dae26eed0d1f9ea4a31f3498d7 used input type number instead